tools/build/node-debian.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-
 set -e
 
-source $NVM_DIR/nvm.sh
+bash $NVM_DIR/nvm.sh
```

Comment:
Any particular reason to use bash? For maximum compatibility, using sh would be nicer (I agree that it does not really matter here)

Reply:
This was an error on my part. Also: This script only works if it's executed with `source`; this means I have to execute the script with bash, which means I may need to tier out the node-debian.sh execution to another level.